cmd/gcp-controller-manager/gcp_config.go

```diff
@@ -88,18 +88,21 @@ func loadGCPConfig(gceConfigPath, gceAPIEndpointOverride string) (gcpConfig, err
 	if gceAPIEndpointOverride != "" {
 		a.Compute.BasePath = gceAPIEndpointOverride
 	}
+	a.Compute.UserAgent = "gcp-controller-manager GCE API client"
```

Comment:
How about just "gcp-controller-manager"?

I'm not sure that distinguishing beta vs ga client will actually have visible value.

Reply:
Makes perfect sense to me. Done.